flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/agg/GroupWindowITCase.scala

```diff
@@ -743,23 +744,27 @@ class GroupWindowITCase extends BatchTestBase {
     )
   }
 
-  @Test(expected = classOf[RuntimeException])
+  @Test
   def testSessionWindowWithProperties(): Unit = {
-    registerCollection(
-      "T",
-      data3WithTimestamp,
-      new RowTypeInfo(INT_TYPE_INFO, LONG_TYPE_INFO, STRING_TYPE_INFO, LOCAL_DATE_TIME),
-      "a, b, c, ts")
-
-    val sqlQuery =
-      "SELECT COUNT(a), " +
-        "SESSION_START(ts, INTERVAL '4' SECOND), " +
-        "SESSION_END(ts, INTERVAL '4' SECOND), " +
-        "SESSION_ROWTIME(ts, INTERVAL '4' SECOND) " +
-        "FROM T " +
-        "GROUP BY SESSION(ts, INTERVAL '4' SECOND)"
 
-    checkResult(sqlQuery, Seq())
+    assertThatThrownBy(
```

Comment:
ditto. Through this method only has one `checkResult`,  we should also wrap the right location.

Reply:
Done.